components/Fly.js

```diff
@@ -1,17 +1,35 @@
 import React from "react";
-import { StyleSheet, Text, View, TouchableOpacity } from "react-native";
+import { StyleSheet, Text, View, TouchableOpacity, Alert } from "react-native";
+import { AntDesign, MaterialCommunityIcons } from "@expo/vector-icons";
+import { deleteFly } from "../ApiCalls";
 
 export default function Fly({ navigation, fly }) {
+  
+  const deletionAlert = (flyId) =>
+    Alert.alert(
+      "Remove Fly",
+      "Are you sure you want to remove this fly from your fly box? Doing so will remove this fly from any fish caught.",
```

Comment:
We could update this message and its wording. Possibly add the fly's name (i.e. "Are you sure you want to remove Pat's Rubber Legs from your fly box?")

Reply:
Nice work with this.  This is a much cleaner way to display a confirmation to delete a fly.